README.md

```diff
@@ -15,6 +15,10 @@ focussing on maintainability, consistency and avoidance of common pitfalls.
 > Error Prone is a static analysis tool for Java that catches common
 > programming mistakes at compile-time.
 
+Read more on how Picnic uses Error Prone (Support) in the blog post [_Picnic
+loves Error Prone: producing high-quality and consistent Java
+code_][picnic-blog-ep-post].
```

Comment:
Replaying my internal debate with the opposite outcome 🙃. But it is at least 5 times more clickable, so let's go for it :)